src/print.rs

```diff
@@ -142,7 +142,13 @@ pub fn module(module_name: &str, args: ArgMatches) {
 }
 
 pub fn get_module(module_name: &str, context: Context) -> Option<String> {
-    modules::handle(module_name, &context).map(|m| m.to_string())
+    Some(
+        handle_module(module_name, &context, &BTreeSet::new())
+            .iter()
+            .map(|m| m.to_string())
+            .collect::<Vec<String>>()
```

Comment:
This allocated a Vec that we immediately drop after joining, it doesn't seem like there is a way in std to join an `Iterator` of `String`.